rest-api-spec/src/main/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -53,49 +53,25 @@ setup:
   - is_true: test_index.mappings
 
 ---
-"Get index infos for mappings only":
-
-  - do:
-      indices.get:
-        index: test_index
-        feature: _mapping
```

Comment:
I'm not clear on why this test was eliminated, because we still support {index}/_mapping by its own without the comma-separated multi values?